src/secrets-kubernetes/src/lib.rs

```diff
@@ -174,3 +174,13 @@ impl SecretsController for KubernetesSecretsController {
         return Ok(());
     }
 }
+
+impl SecretsReader for KubernetesSecretsController {
+    fn read(&self, _id: mz_repr::GlobalId) -> Result<Vec<u8>, Error> {
+        unreachable!()
```

Comment:
The details of implementation for the KubernetesSecretsController are being handled by @mkysel, I forged ahead with the FilesystemSecretsController implementation because it was sufficient to make testdrive pass and will not be used in production.

Reply:
might make sense to leave a TODO comment or something